src/pf-utils/pf-utils.js

```diff
@@ -38,4 +38,76 @@ let I18nUtil = function() {
     self.mixin = mixin;
   };
 };
-export let i18n = new I18nUtil();
+let i18n = new I18nUtil();
+export {i18n};
+
+/**
+ * --------------------------------------------------------------------------
+ * PfUtil
+ * Native Javascript for Bootstrap 3 | Internal Utility Functions
+ * --------------------------------------------------------------------------
+ */
+
+class PfUtil {
```

Comment:
I don't feel strongly about the "native" name, but thinking separate files is probably best.